### PR TITLE
add dfs tests for https://github.com/tc39/ecma262/pull/1669

### PR DESCRIPTION
**Pull request by devsnek**
cc @guybedford

### CODE REVIEW

File: test/language/module-code/verify-dfs-a_FIXTURE.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { evaluated, check } from './verify-dfs.js';
+
+check(import('./verify-dfs-b_FIXTURE.js'));
```
Comment (leobalter) [inline test/language/module-code/verify-dfs-a_FIXTURE.js]: nice!

File: test/language/module-code/verify-dfs.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  evaluated.order.push(name);
+}
+
+export function check(promise) {
+  promise.then(() => {
+    assert.sameValue(evaluated.order.length, 2);
+    assert.sameValue(evaluated.order[0], 'A');
+    assert.sameValue(evaluated.order[1], 'B');
+  })
+  .then($DONE, $DONE);
```
Comment (leobalter) [inline test/language/module-code/verify-dfs.js]: I like it a lot

File: test/language/module-code/verify-dfs.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/*---
```
Comment (leobalter) [inline test/language/module-code/verify-dfs.js]: we need a copyright header in each file (including the fixtures) for the lint to pass. This requirement is an general rule I'd love to remove but we can't change it for now.

**Comment by devsnek** [comment]
bump

**Pull request merged**